cmd/utils/nodecmd/accountcmd.go

```diff
@@ -146,14 +142,96 @@ The account is saved in encrypted format, you are prompted for a passphrase.
 
 You must remember this passphrase to unlock your account in the future.
 
-For non-interactive use the passphrase can be specified with the -password flag:
-
-    klay account import [options] <keyfile>
+For non-interactive use the passphrase can be specified with the --password flag:
 
-Note:
-As you can directly copy your encrypted accounts to another klay instance,
+Note, as you can directly copy your encrypted accounts to another klay instance,
 this import mechanism is not needed when you transfer an account between
 nodes.
+`,
+		},
+		{
+			Name:   "bls-info",
+			Usage:  "Calculate BLS public key info",
+			Action: accountBlsInfo,
+			Flags: []cli.Flag{
+				utils.NodeKeyFileFlag,
+				utils.NodeKeyHexFlag,
+				utils.BlsNodeKeyFileFlag,
+				utils.BlsNodeKeyHexFlag,
+				utils.BlsNodeKeystoreFileFlag,
+				utils.PasswordFileFlag,
+			},
+			Description: `
+Calculate BLS public key info (the public key and proof-of-possession)
+then prints to STDOUT.
+
+The input must be one of the following:
+
+(1) A 32-byte raw EC private key (--nodekey, --nodekeyhex)
+    In this case the EC private key is first derived to a BLS private key,
+    then the BLS public key info is calculated.
+(2) A 32-byte raw BLS private key (--bls-nodekey, --bls-nodekeyhex)
+(3) An encrypted BLS keystore JSON (--bls-nodekeystore)
+
+EXAMPLES
+
+# Print public key info of the BLS key derived from the EC nodekey
+kcn acount bls-info --nodekey /var/kcnd/data/nodekey
```

Comment:
```suggestion
kcn account bls-info --nodekey /var/kcnd/data/nodekey
```

There are two more typos below.

Reply:
Fixed in f4d41fd9.